### PR TITLE
change turbo jpeg to fetch partial decoding version

### PR DESCRIPTION
**Pull request by LakshmiKumar23**

### CODE REVIEW

**Comment by LakshmiKumar23** [comment]
@kiritigowda ready to be merged. Builds successfully

File: MIVisionX-setup.py [hunk 1]
```diff
@@ -222,9 +222,8 @@
 			os.system('sudo -v')
 			os.system('sudo '+linuxFlag+' '+linuxSystemInstall+' -y '+linuxSystemInstall_check+' install libboost-all-dev clang')
 			#turbo-JPEG
-			os.system('(cd '+deps_dir+'; wget https://downloads.sourceforge.net/libjpeg-turbo/libjpeg-turbo-2.0.3.tar.gz )')
-			os.system('(cd '+deps_dir+'; tar xf libjpeg-turbo-2.0.3.tar.gz )')
-			os.system('(cd '+deps_dir+'/libjpeg-turbo-2.0.3; mkdir build; cd build; '+linuxCMake+' -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=RELEASE -DENABLE_STATIC=FALSE -DCMAKE_INSTALL_DOCDIR=/usr/share/doc/libjpeg-turbo-2.0.3 -DCMAKE_INSTALL_DEFAULT_LIBDIR=lib ..; make -j 4; sudo make install )')
+			os.system('(cd '+deps_dir+'; git clone https://github.com/Indumathi31/libjpeg-turbo.git )')
```
Comment (kiritigowda) [inline MIVisionX-setup.py:225]: can we get a release or something constant? Master branch can be updated.

Comment (kiritigowda) [reply MIVisionX-setup.py:225]: aby update on this @LakshmiKumar23 ?

Comment (LakshmiKumar23) [reply MIVisionX-setup.py:225]: @kiritigowda Spoke to MCW about this today. They will give me us a release version in a day

Comment (LakshmiKumar23) [reply MIVisionX-setup.py]: @kiritigowda changed to release version 2.0.4. Please verify

**Comment by kiritigowda** [comment]
@LakshmiKumar23 @rrawther this PR will be merged after PR #328

**Pull request merged**